MailCore/Models/Message.swift

```diff
@@ -323,28 +318,13 @@ public final class Message: Object, Decodable, Identifiable {
             self.date = date
         } else {
             date = Date()
-            let dateParsingError = Event(level: .error)
-
-            let breadcrumb = SentryDebug.createBreadcrumb(
-                level: .warning,
-                category: "Thread algo",
-                message: "Nil message date decoded",
-                data: ["uid": uid]
-            )
-
-            SentrySDK.capture(error: ErrorDomain.dateParsingFailed) { scope in
-                if let date = try? values.decode(String.self, forKey: .date) {
-                    scope.setExtra(value: date, key: "dateString")
-                }
-
-                if let date = try? values.decode(Data.self, forKey: .date) {
-                    scope.setExtra(value: date, key: "dateRaw")
-                }
-
-                scope.addBreadcrumb(breadcrumb)
-            }
-
-            SentrySDK.addBreadcrumb(breadcrumb)
+            SentrySDK
+                .addBreadcrumb(SentryDebug.createBreadcrumb(
+                    level: .warning,
+                    category: "Thread algo",
+                    message: "Nil message date decoded",
+                    data: ["uid": uid]
+                ))
```

Comment:
Could you move those lines in SentryDebug.swift, it improves readability to have all of them in one place

Reply:
Allright, I refactored to remove the Sentry dependency from Message.swift.